lib/index.js

```diff
@@ -126,22 +144,22 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       let data = _getSessionAuthenticationData({req});
       const authenticatedMethods = data.authenticatedMethods || [];
 
+      // get clientId from cookie
+      const {name} = cfg.cookies.clientId;
+      const clientId = req.cookies[name];
+
       // make sure session data includes whether or not token client has
       // registered
-      if(!authenticatedMethods.includes('token-client-registration')) {
-        // get clientId from cookie
-        const {name} = cfg.cookies.clientId;
-        const clientId = req.cookies[name];
-        if(clientId) {
-          const result = await brAuthnToken.clients.isRegistered(
-            {actor: null, email, account, clientId});
-          if(result.registered) {
-            authenticatedMethods.push('token-client-registration');
-          }
+      if(!authenticatedMethods
+        .includes('token-client-registration' && clientId)) {
```

Comment:
It looks like there maybe a misplaced closing parenthesis here.  I assume this should be:
```suggestion
      if(!authenticatedMethods.includes('token-client-registration') && clientId) {
```